tests/test_helpers/test_env_vars.py

```diff
@@ -78,3 +78,10 @@ def test_convert_value_str_to_type():
     assert convert_value_str_to_type("none", "str") is None
     assert convert_value_str_to_type("null", "str") is None
     assert convert_value_str_to_type("None", "str") is None
+
+
+def test_convert_list_string_to_type():
+    """Test convert_string lists to list."""
+    assert convert_value_str_to_type(str(['a']), "list") == ['a']
```

Comment:
I dont think this test should fail.

Reply:
```
>>> import json
>>> str(['a'])
"['a']"
>>> json.dumps(['a'])
'["a"]'
>>> 
```
The assumption is that the str is json encoded for list and dict (as we can see from the implementation)

Reply:
Then i can call json.dumps on the base_deployment yes?

Reply:
Nothing needs calling anywhere. The yaml needs to specify the correctly formatted values.